public/css/restaurant-home.css

```diff
@@ -0,0 +1,150 @@
+@import url('https://framer-ui.netlify.app/components/framer.css');
+
+:root{
+    --navbar-bg:#F2F2F2;
+    --food-slider-bg:#171A29;
+    --text-col-white:#FFFFFF;
+    --text-col-gray:#9EA0A6;
+    --radio-color:#4f46e5;
+}
+.navbar{
+    padding: 1rem 1rem 0;
+}
+.nav-zindex{
+    z-index: 999;
+}
+
+.discount-container{
+    height: 35vh;
+    background: var(--food-slider-bg);
+    position: relative;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    gap: 2rem;
+}
+
+
+
+.hotel-img{
+    height: 15.625rem;
+    width: 20.625rem;
+    border-radius: 7px;
```

Comment:
Use rems instead of pxs

Reply:
@varun13169 if i am not wrong Vikram told in his session that if the sizing value is more than 10px then only use rems